src/views/statutory/Edit.vue

```diff
@@ -89,6 +89,18 @@
           Please select the event type wisely, as it will influence the event's workflow. <strong>It can't be changed later.</strong>
         </div>
 
+        <div class="field" v-if="!$route.params.id">
+          <label class="label">Is the event going to be fully vegetarian?</label>
+          <div class="control">
+            <input type="checkbox" data-cy="vegetarian" v-model="vegetarian" />
+            <label class="checkbox"> Yes, the event will be without any meat</label>
+          </div>
+        </div>
+
+        <div class="notification is-success" v-if="!$route.params.id">
+          Consider if you want to make the event fully vegetarian / vegan! <strong>It can't be changed later.</strong>
```

Comment:
Would be nice to able to change it

Reply:
Would that also mean sending all already subscribed users an email that they can / should change their dietary preferences?

Reply:
No, just that when new people apply or people update their application also get the option to choose meat-eater. Nothing special

Reply:
And what if the event was non-vegetarian, people apply as meat-eater and then it becomes vegetarian?

Reply:
Oh yes, that's a good one. I forgot about that scenario. In that case it might be nice to set all current applications to 'vegetarian' and send an email to the changed applications informing them that the event is now vegetarian and their preferences have been updated accordingly. But let's make that a different story then